src/data-storage/domains.js

```diff
@@ -0,0 +1,11 @@
+export const domains = [
```

Comment:
please remove this file for now, and we shouldn't be using `javascript` files in this project. thanks man.